cfpb/consumerfinance.gov
# More icon conversions to use our standard loader

## Changes

- FIG: use standard loader for print icon.
- eregs: Use standard loader for next/prev buttons.
- TDP: Use standard loader for icons in activities search and details. Also, add missing list classes.
- YES: Use standard loader for icons in car buying guide.


## How to test this PR

1. Not sure how to test the FIG @contolini 
1. Visit a regs page like http://localhost:8000/rules-policy/regulations/1024/4/#a-1-ii and see next/prev page arrows are okay.
2. Visit http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/ and see that the building block icons are okay. Click on an activity and see the building block icons are okay in the sidebar.
3. Visit http://localhost:8000/consumer-tools/educator-tools/buying-a-car/ and see that the error icon is okay when attempting to print the form without an check boxes checked.


## Screenshots

<img width="714" alt="Screen Shot 2023-03-21 at 7 27 17 PM" src="https://user-images.githubusercontent.com/704760/226764893-53ab7a88-5866-4a39-a2b8-375a4d943571.png">

<img width="277" alt="Screen Shot 2023-03-21 at 7 29 12 PM" src="https://user-images.githubusercontent.com/704760/226764894-15ad92c1-1be1-4684-9e3e-22f4ff5c4438.png">

<img width="391" alt="Screen Shot 2023-03-21 at 7 29 17 PM" src="https://user-images.githubusercontent.com/704760/226764896-10accd64-9dcd-4acf-8998-41af9d237509.png">

<img width="296" alt="Screen Shot 2023-03-21 at 7 29 28 PM" src="https://user-images.githubusercontent.com/704760/226764897-7e8f943a-73d7-4852-9f5e-e80bba6f6517.png">
